nativeruntime/cpp/androidfw/CursorWindow.cpp

```diff
@@ -39,14 +42,49 @@ CursorWindow::CursorWindow(const String8& name, int ashmemFd, void* data,
       mData(data),
       mSize(size),
       mReadOnly(readOnly) {
+  // mHeader = static_cast<Header*>(mData);
+  #if defined(_WIN32)
+  mHeader = new Header;
+  #else
   mHeader = static_cast<Header*>(mData);
+  #endif
 }
 
 CursorWindow::~CursorWindow() {
+  // ::munmap(mData, mSize);
```

Comment:
nit: delete these comments

Reply:
Okay, I will delete it at next commit, and squash all commits to one.

Reply:
Should commented include line by this PR removed too?

Reply:
Deleted it and squashed all commits to one commit.